### PR TITLE
cuda-api-wrappers: Added versions  0.6.9 and 0.7.0-b3

### PR DESCRIPTION
**Pull request by eyalroz**
Specify library name and version:  **cuda-api-wrappers/0.6.9**

New version of the library is out; plus a new beta of an upcoming major version, that's based on the new version being released now. No new dependencies.

---

- [X] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [X] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6). *<- I've only added two version entries*
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated. *<- I'm not exactly sure what I'm supposed to try.*

### CODE REVIEW

File: recipes/cuda-api-wrappers/config.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 versions:
-  "0.7.0-b2":
+  "0.7.0-b3":
     folder: all
```
Comment (valgur) [inline recipes/cuda-api-wrappers/config.yml:3]:
```suggestion
  "0.7.0-b3":
    folder: all
  "0.7.0-b2":
    folder: all
```

**Review by valgur — CHANGES_REQUESTED** [review]
A small error in config.yml, LGTM otherwise.

File: recipes/cuda-api-wrappers/config.yml (hunk 1 above)
Comment (eyalroz) [reply recipes/cuda-api-wrappers/config.yml]: Sorry, fixed this.

**Comment by valgur** [comment]
Can be closed in favor of #25029, probably.

**Pull request closed without merging**